java/core/src/java/org/apache/orc/impl/PhysicalFsWriter.java

```diff
@@ -129,6 +124,18 @@ public PhysicalFsWriter(FileSystem fs,
     }
   }
 
+  public PhysicalFsWriter(FileSystem fs,
```

Comment:
Please add this before the original constructor.

Reply:
@dongjoon-hyun done